docs/deploy/subnet_discovery.md

```diff
@@ -1,7 +1,12 @@
 # Subnet Auto Discovery
 AWS Load Balancer controller auto discovers network subnets for ALB or NLB by default. ALB requires at least two subnets across Availability Zones, NLB requires one subnet.
 The subnets must be tagged appropriately for the auto discovery to work. The controller chooses one subnet from each Availability Zone. In case of multiple tagged subnets in
-an Availability Zone, the controller will choose the first one in lexicographical order by the Subnet IDs. If you use `eksctl` or an Amazon EKS AWS CloudFormation template to
+an Availability Zone, the controller will select subnets based on the count of available IP addresses. For ALB, each Availability Zone subnet has at least 8 free IP addresses; 
```

Comment:
In case of multiple subnets, the auto-discovery still follows the lexicographic order, limited to the subnets with at least 8 IP addresses. 

Description is a bit long. See if we can summarize it further for clarity.

Reply:
Thanks, has made changes accordingly.